CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 2.2.0
```

Comment:
As a drive by comment, using a `-wip` suffix here lets people know that the content in the repo is not yet published.

Reply:
Noted

Reply:
Actually I think the plan was to publish immediately. But I got distracted...